### PR TITLE
Fix text selection on touch systems

### PR DESCRIPTION
**Pull request by kswenson**
Fix touch text selection [#158700634]
- disable touch text selection at the page level
- enable it in components where it's appropriate

@jsandoe This PR depends on #230. Once that is merged I will rebase this one.

### CODE REVIEW

File: apps/dg/resources/dg.css [hunk 1]
```diff
@@ -42,6 +42,30 @@
...
+    -ms-user-select: none;
+    user-select: none;
+    -webkit-touch-callout: none;
+}
+
+/* SproutCore inexplicably left out -ms-user-select and -webkit-user-select */
+.sc-view.allow-select {
+    -khtml-user-select: text;
+    -moz-user-select: text;
+    -ms-user-select: none;
```
Comment (jsandoe) [inline apps/dg/resources/dg.css]: Why "none" for ms-user-select? Do we not want to allow selection there?

Comment (kswenson) [reply apps/dg/resources/dg.css]: Good catch. Copy/paste error.

Comment (jsandoe) [inline apps/dg/resources/dg.css]: Why "none" for ms-user-select? Do we not want to allow selection there?

Comment (kswenson) [reply apps/dg/resources/dg.css]: Good catch. Copy/paste error.

**Pull request merged**